src/v2i-hub/MUSTSensorDriverPlugin/src/MUSTSensorDetection.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+#include <stdexcept>      // std::out_of_range
```

Comment:
Not sure this comment here is helpful.

Reply:
I do not think this is worth changing. It is meant to indicate why this is imported which is valuable because C++ nor VSCode have good tools for auditing imported headers for use.